go/test/endtoend/reparent/utils/utils.go

```diff
@@ -229,10 +229,10 @@ func StartNewVTTablet(t *testing.T, clusterInstance *cluster.LocalProcessCluster
 		require.FailNow(t, "Error starting mysql: %s", err.Error())
 	}
 
+	// The tablet should come up as serving since the primary for the shard already exists
+	tablet.VttabletProcess.ServingStatus = "SERVING"
 	err = tablet.VttabletProcess.Setup()
 	require.NoError(t, err)
-	err = tablet.VttabletProcess.WaitForTabletStatuses([]string{"SERVING", "NOT_SERVING"})
```

Comment:
Why remove the wait, shouldn't it be SERVING?

Reply:
There is a wait in `Setup` too, which is controlled by the `tablet.VttabletProcess.ServingStatus` property. This is why I set the property to `SERVING` a couple of lines earlier and removed the wait here entirely. It is redundant.